code/game/machinery/computer/_computer.dm

```diff
@@ -21,6 +21,7 @@
 	var/authenticated = FALSE
 
 /datum/armor/machinery_computer
+	bullet = -50
```

Comment:
does this like, work?

Reply:
I don't think I accounted for negative armour when doing armour penetration calculation changes. I should probably make sure that armour penetration won't reduce negative armour.